### PR TITLE
[ci] Add consistency tests

### PR DESCRIPTION
**Pull request by daamien**
Checks if plain mode and binary mode provide the same result

### CODE REVIEW

File: t/consistency.t [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+BIN=$BATS_TMPDIR/out.bin
+OUT=$BATS_TMPDIR/out.json
+OUT_FROM_BIN=$BATS_TMPDIR/out_from_bin.json
+./pgbadger $IN -o $OUT
+./pgbadger $IN  -o $BIN
+./pgbadger $BIN --format=binary -o $OUT_FROM_BIN
+}
+
+@test "Consistent count" {
+    # Assert out.html is not created in PWD.
```
Comment (bersace) [inline t/consistency.t]: @daamien c'est du copié collé ça, non ?

File: t/consistency.t [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    N=$(jq .database_info.postgres.count < $OUT)
+    test $N -eq 629
+    N=$(jq .database_info.postgres.count < $OUT_FROM_BIN)
+    test $N -eq 629
+}
+
+
+@test "Consistent query_total" {
+    # Assert out.html is not created in PWD.
```
Comment (bersace) [inline t/consistency.t]: @daamien ;-)

File: t/consistency.t [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    N=$(jq .overall_stat.histogram.query_total < $OUT)
+    test $N -eq 629
+    N=$(jq .overall_stat.histogram.query_total < $OUT_FROM_BIN)
+    test $N -eq 629
+}
+
+
+@test "Consistent peak.write" {
+    # Assert out.html is not created in PWD.
```
Comment (bersace) [inline t/consistency.t]: ☝️

File: t/consistency.t [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    N=$(jq '.overall_stat.peak."2017-09-06 08:49:19".write' < $OUT)
+    test $N -eq 1
+    N=$(jq '.overall_stat.peak."2017-09-06 08:49:19".write' < $OUT_FROM_BIN)
+    test $N -eq 1
+}
```
Comment (bersace) [inline t/consistency.t]: `\n`

**Comment by daamien** [comment]
😅 

Sorry I felt in a hurry to push this last night. Should have waited a few hours and read it  with a clear mind 😞

File: t/consistency.t [hunk 5]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+# -*- shell-script -*-
+
+setup() {
+IN=t/fixtures/light.postgres.log.bz2
```
Comment (bersace) [inline t/consistency.t]: Shouldn't this be indented ?

**Pull request merged**